tests/integration/common.py

```diff
@@ -12,6 +12,22 @@ class ExitCode(IntEnum):
     INCORRECT_USAGE = 2
 
 
+class Output:
+    command: list[Path | str]
+    stdout: str
+    stderr: str
+    returncode: int
+
+    def __init__(self, command: list[Path | str], stdout: str, stderr: str, returncode: int):
+        self.command = command
+        self.stdout = stdout
+        self.stderr = stderr
+        self.returncode = returncode
+
+    def __str__(self) -> str:
+        return f"command: {self.command}"
```

Comment:
I don't particularly mind, but we don't use this yet, right?

Reply:
We don't use it directly , but I think it is handy to have while debugging the test to check what is the output ( which I've personally have done in this PR ) that's why I think it would be nice to have in our dev toolchain